yarn-project/simulator/src/client/simulator.test.ts

```diff
@@ -5,7 +5,7 @@ import { type FunctionArtifact, getFunctionArtifact } from '@aztec/foundation/ab
 import { AztecAddress } from '@aztec/foundation/aztec-address';
 import { poseidon2Hash } from '@aztec/foundation/crypto';
 import { Fr, type Point } from '@aztec/foundation/fields';
-import { TokenContractArtifact } from '@aztec/noir-contracts.js/Token';
+import { TokenBlacklistContractArtifact } from '@aztec/noir-contracts.js';
```

Comment:
Did this change here because TokenContract can no longer be used with the standard `computeNoteHidingPoint` func.